<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pc</class>
 <widget class="QDialog" name="pc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1155</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;

</string>
  </property>
  <widget class="QLabel" name="label_38">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>430</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Giacomo Guilizzoni</string>
   </property>
  </widget>
  <widget class="QPushButton" name="readOut">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>131</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_58">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>470</y>
     <width>27</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>500</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Agence</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_43">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>500</y>
     <width>48</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>15</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_57">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>501</y>
     <width>48</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>7 300</string>
   </property>
  </widget>
  <widget class="QComboBox" name="agence">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>191</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>1141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="cal">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>240</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_48">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>570</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Paris</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_59">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>372</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Volume</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_37">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>372</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Volume</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_53">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>530</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Quentin</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_50">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>430</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Peldi</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_49">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>500</y>
     <width>48</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Pierre</string>
   </property>
  </widget>
  <widget class="QPushButton" name="download">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>70</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="home">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
color:rgb(50, 0, 0);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="print">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>70</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>20</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(78, 214, 255);
font: 63 italic 14pt &quot;Source Code Pro Semibold&quot;;</string>
   </property>
   <property name="text">
    <string>La Banque</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>1141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_40">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>465</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Marco Botton</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_44">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>530</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Christophe Foster</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>250</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Date</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_41">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>468</y>
     <width>27</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_45">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>534</y>
     <width>33</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QLabel" name="date">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>240</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
background-color:rgb(220, 220, 220);</string>
   </property>
   <property name="text">
    <string>13/05/2020 - 13/06/2020</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_46">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>570</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Ratatouille Jacques</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_51">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>371</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Client</string>
   </property>
  </widget>
  <widget class="QPushButton" name="icon1">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_52">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>470</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Larbi</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>340</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Assurance Pc</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_56">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>535</y>
     <width>33</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>1 260</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_42">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>496</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Valerie Liberty</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_47">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>570</y>
     <width>27</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>120</y>
     <width>481</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;OpenSymbol&quot;;</string>
   </property>
   <property name="text">
    <string>Produits d'assurances - &gt; PC</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_39">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>436</y>
     <width>33</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_55">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>430</y>
     <width>33</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>1 200</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>360</y>
     <width>441</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="mode">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>11</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(158, 158, 158);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>10</y>
     <width>21</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(170, 0, 0);
font: 195 22pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>B</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_36">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>371</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Nom de l'employee</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_54">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>570</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>3 402</string>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>372</y>
     <width>20</width>
     <height>241</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>372</y>
     <width>20</width>
     <height>241</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>372</y>
     <width>20</width>
     <height>241</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_6">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>400</y>
     <width>421</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="file">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="view">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>80</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="edit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="help">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>80</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendar">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>170</y>
     <width>312</width>
     <height>183</height>
    </rect>
   </property>
  </widget>
  <widget class="QtCharts::QChartView" name="assurancePC">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>371</y>
     <width>301</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <zorder>home</zorder>
  <zorder>tableView</zorder>
  <zorder>label_38</zorder>
  <zorder>readOut</zorder>
  <zorder>label_58</zorder>
  <zorder>label_7</zorder>
  <zorder>label_43</zorder>
  <zorder>label_57</zorder>
  <zorder>agence</zorder>
  <zorder>line_2</zorder>
  <zorder>cal</zorder>
  <zorder>label_48</zorder>
  <zorder>label_59</zorder>
  <zorder>label_37</zorder>
  <zorder>label_53</zorder>
  <zorder>label_50</zorder>
  <zorder>label_49</zorder>
  <zorder>download</zorder>
  <zorder>print</zorder>
  <zorder>label_2</zorder>
  <zorder>line</zorder>
  <zorder>label_40</zorder>
  <zorder>label_44</zorder>
  <zorder>label_8</zorder>
  <zorder>label_41</zorder>
  <zorder>label_45</zorder>
  <zorder>date</zorder>
  <zorder>label_46</zorder>
  <zorder>label_51</zorder>
  <zorder>icon1</zorder>
  <zorder>label_52</zorder>
  <zorder>label_10</zorder>
  <zorder>label_56</zorder>
  <zorder>label_42</zorder>
  <zorder>label_47</zorder>
  <zorder>label_9</zorder>
  <zorder>label_39</zorder>
  <zorder>label_55</zorder>
  <zorder>mode</zorder>
  <zorder>label_3</zorder>
  <zorder>label_36</zorder>
  <zorder>label_54</zorder>
  <zorder>line_3</zorder>
  <zorder>line_4</zorder>
  <zorder>line_5</zorder>
  <zorder>line_6</zorder>
  <zorder>file</zorder>
  <zorder>view</zorder>
  <zorder>edit</zorder>
  <zorder>help</zorder>
  <zorder>calendar</zorder>
  <zorder>assurancePC</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtCharts::QChartView</class>
   <extends>QGraphicsView</extends>
   <header location="global">QtCharts/QChartView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
